### ISSUE TITLE
Regular Expression Denial of Service in vinyl-fs > glob-stream > micromatch > braces

### ISSUE THREAD

**Issue by Ambient-Impact** [SassDoc/sassdoc#537]
Hi there. Just ran ```npm audit``` and got this:

```
  Low             Regular Expression Denial of Service

  Package         braces

  Patched in      >=2.3.1

  Dependency of   grunt-sassdoc [dev]

  Path            grunt-sassdoc > sassdoc > vinyl-fs > glob-stream >
                  micromatch > braces

  More info       https://npmjs.com/advisories/786
```

It looks like newer versions of [```glob-stream```](https://github.com/gulpjs/glob-stream/blob/master/package.json) and thus [```vinyl-fs```](https://github.com/gulpjs/vinyl-fs/blob/master/package.json) are no longer using ```micromatch```, so upgrading would fix this warning.

**Comment by joekrump** [comment]
It appears that an update to version 2.3.2 (the first version that contains the security fix) should be possible without running into breaking changes, and therefore this should be a pretty trivial update. https://github.com/micromatch/braces/blob/master/CHANGELOG.md#232---2018-04-08

**Comment by joekrump** [comment]
I've opened a pull request to update the version of `vinyl-fs` here: https://github.com/SassDoc/sassdoc/pull/547

**Comment by joekrump** [comment]
This is issue can now be closed I think as version 2.7.2 has now been released and contains the fix.
